### ISSUE TITLE
identify loci in catalog but not in sumstats

### ISSUE THREAD

**Issue by jdhayhurst** [EBISPOT/goci#250]
For all the SNPs in the Catalog what % do we have the SS?

Check against the subset that is in the db of in standard format.

**Comment by jdhayhurst** [comment]
125773 snps in catalog (matching the "rs[0-9]+$" pattern).  There are 1006 of these not present in the summary stats files that are available in the sumstats API (a subset of the total sumstats). So <0.8% of the snps in the catalog are not represented in the summary statistics.

**Comment by jdhayhurst** [comment]
I filtered GWAS Catalog associations for those that are single variants and have their trait represented in the summary stats API, leaving 97,977 associations (I also exploded traits, where associations belonged to multiple traits). I then utilised the summary statistics API to identify any other associations within 100kb +- window for same trait for all remaining associations.

Results:
64,114 of the 97,977 lead associations that I checked against the summary stats API had one or more signals within 100kb and p-value < 1E-5.

Here is an example for the single association seen here: https://www.ebi.ac.uk/gwas/variants/rs1882961:
- Take the genomic location, 12:15184047 and create a query specifying upper and lower bounds for the region using the summary stats api:
  https://www.ebi.ac.uk/gwas/summary-statistics/api/chromosomes/21/associations?bp_lower=15084047&bp_upper=15284047
- add parameters to filter by trait, EFO_0006335, and add a p-value upper limit (I set this to 1E-5)
  https://www.ebi.ac.uk/gwas/summary-statistics/api/chromosomes/21/associations?bp_lower=15084047&bp_upper=15284047&trait=EFO_0006335&&p_upper=0.00001
- So that's the query, which if you visit will return a json payload showing you one association at 21:15184047 from a different study, GCST006259, for the same trait. If there wasn't anything that matches that query, it would tell you that nothing satisfied the query.
